---
float32 communication_error
float32 motor_error
float32 current_position
float32 current_speed
float32 current_force
float32 current_voltage
float32 current_temperature
bool completed
bool calibrated
